src/Components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styles from "./LoginForm.module.css";
+
+function LoginForm() {
+  return (
+    <form className={styles.form}>
+      <div className={styles.input}>
+        <input type="text" placeholder="email" className={styles.email} />
+        <input
+          type="password"
+          placeholder="password"
+          className={styles.password}
+        />
+      </div>
+      <div className={styles.submit}>
+        <span className={styles.logintext}>login</span>
+        <input type="submit" value="→" className={styles.login} />
+      </div>
```

Comment:
I'd consider labeling the input element instead of using a span next to it. That way, "login" would also be clickable.
You might have to adjust the CSS if you do that. 

```suggestion
      <label className={styles.submit}>
        login
        <input type="submit" value="→" className={styles.login} />
      </label>
```